clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -2849,6 +2850,20 @@ public void testUnusedConfigs() {
         }
     }
 
+    @Test
+    public void testUnknownConfigs() {
```

Comment:
I think we should also test the mix of unknown and unused configs case, because we have some logic for them, right?

Reply:
In fact, I don't recommend changing the current logic of the unused method, because for KafkaClient, unknownKeys also belong to unusedKeys, what do you think?
Of course, I would add mixed test cases.

Reply:
I think we can add the mix of unknown and unused test case, no matter we want to change unused logics or not. That is, like the previous `testUnusedConfigs` test, we can add an unknown configs into it, and verify the unUsed and unknown results are as what we expected.

Reply:
Sure,I will add.